src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -226,6 +226,16 @@ public interface IDurableOrchestrationClient
         /// <returns>A task that completes when the terminate message is enqueued if necessary.</returns>
         Task TerminateAsync(string instanceId, string reason);
 
+        /// <summary>
+        /// Suspends a running orchestration instance.
+        /// </summary>
+        Task SuspendAsync(string instanceId, string reason);
+
+        /// <summary>
+        /// Resumes a suspended orchestration instance.
+        /// </summary>
+        Task ResumeAsync(string instanceId, string reason);
+
```

Comment:
Are we currently doing anything with the `reason` parameter? It would be good to incorporate it in a similar fashion to the other management APIs

Reply:
What do you mean to incorporate it into other management APIs?

Reply:
The management APIs are: suspend, resume, terminate, startNewOrchestrationAsync, etc.
I meant that we should ensure that the `reason` parameter is used and shown to the user when they inspect the status of their suspended orchestrator. I know that an earlier implementation of this feature didn't do that, so I wrote this comment to make sure that the current version indeed _used_ the `reason` parameter.

After reviewing the code, I'm fairly certain this is being used, so I'll resolve my own comment.